dotnet/src/Functions/Functions.OpenApi/Model/RestApiOperation.cs

```diff
@@ -250,8 +250,9 @@ private string BuildPath(string pathTemplate, IDictionary<string, object?> argum
     /// </summary>
     /// <param name="serverUrlOverride">Override for REST API operation server url.</param>
     /// <param name="apiHostUrl">The URL of REST API host.</param>
+    /// <param name="arguments">The operation arguments.</param>
     /// <returns>The operation server url.</returns>
-    private Uri GetServerUrl(Uri? serverUrlOverride, Uri? apiHostUrl)
+    private Uri GetServerUrl(Uri? serverUrlOverride, Uri? apiHostUrl, IDictionary<string, object?> arguments)
```

Comment:
```suggestion
    private Uri GetServerUrl(Uri? serverUrlOverride, Uri? apiHostUrl, IDictionary<string, string> arguments)
```

The server value type is supposed to be a string

Reply:
The argument dictionary we use allows objects as these are used elsewhere for substitutions